modules/diskq/logqueue-disk.c

```diff
@@ -67,7 +68,13 @@ log_queue_disk_start(LogQueue *s)
   g_assert(!qdisk_started(self->qdisk));
   g_assert(self->start);
 
-  return self->start(self);
+  if (self->start(self))
+    {
+      log_queue_queued_messages_add(s, log_queue_get_length(s));
```

Comment:
Is this a bugfix?

Reply:
No, the same task was done by `log_queue_register_stats_counters()` before the PR, now I had to move it here.

It is explained in a bit more details in the commit msg:
> If a LogQueue implementation does something related to these, they must update them themselves. A new FIFO instance is always empty, so there is nothing to do there. A new disk-buffer instance might start with an existing disk-queue file. In log_queue_disk_start() we update the queued counter with get_length(), and the non-reliable disk-buffer updates the memory_usage counter during its start().

Reply:
Oops, got it. Thanks.